lib/build.js

```diff
@@ -443,13 +443,14 @@ function createWebpackMiddleware () {
   const clientCompiler = webpack(clientConfig)
   this.clientCompiler = clientCompiler
   // Add the middleware to the instance context
-  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, {
+  this.webpackDevMiddleware = pify(require('webpack-dev-middleware')(clientCompiler, Object.assign(clientConfig.devServer||{},{
```

Comment:
Is `devServer` a standard option for client config? [here](https://github.com/webpack/webpack-dev-middleware) seems using an extra object for devMiddleware so i would suggest using `this.options.build.devMiddleware` instead. (Also adding it in `lib/core/options.js`) Also It would better using `Object.assing({}, this.options, ...)` to prevent mutate original options.